byzcoin/contract_deferred.go

```diff
@@ -43,6 +43,29 @@ type DeferredData struct {
 	ExecResult [][]byte
 }
 
+// String returns a human readable string representation of the deferred data
+func (dd DeferredData) String() string {
```

Comment:
Performance for this particular use case does not matter, but string concatenation like this in Go causes lots of copying and garbage creation. It would be better to use https://godoc.org/strings#Builder.

Reply:
Thanks for the tip! 👍   (I based it on what I found in `byzcoin/transaction.go:222`)